jinahub/encoders/text/DPRTextEncoder/README.md

```diff
@@ -0,0 +1,99 @@
+# ✨ DPRTextEncoder
+
+ **DPRTextEncoder** is a class that encodes text into embeddings using the DPR model from huggingface transformers.
+
+The **DPR** model was originally proposed in [Dense Passage Retrieval for Open-Domain Question Answering](https://arxiv.org/abs/2004.04906).
+
+This encoder supports both the DPR context and question encoders - you should specify which type you are using with the `encoder_type` parameter.
+
+The following parameters can be passed on initialization:
+
+- `pretrained_model_name_or_path`: Can be either:
+	- the model id of a pretrained model hosted inside a model repo
+		on huggingface.co.
+	- A path to a directory containing model weights, saved using
+		the transformers model's `save_pretrained()` method
+- `encoder_type`: Either `'context'` or `'question'`. Make sure this
+	matches the model that you are using.
+- `base_tokenizer_model`: Base tokenizer model. The possible values are
+	the same as for the ``pretrained_model_name_or_path`` parameters. If not
+	provided, the ``pretrained_model_name_or_path`` parameter value will be used
+- `max_length`: Max length argument for the tokenizer
+- `default_batch_size`: Default batch size for encoding, used if the
+	batch size is not passed as a parameter with the request.
+- `default_traversal_paths`: Default traversal paths for encoding, used if the
+	traversal path is not passed as a parameter with the request.
+- `device`: The device (cpu or gpu) that the model should be on.
+
+
+**Table of Contents**
+
+- [🌱 Prerequisites](#-prerequisites)
+- [🚀 Usages](#-usages)
+- [🎉️ Example](#%EF%B8%8F-example)
+- [🔍️ Reference](#%EF%B8%8F-reference)
+
+## 🌱 Prerequisites
+
+No prerequisites are required to run this executor.
+
+## 🚀 Usages
+
+### 🚚 Via JinaHub
+
+Use the prebuilt images from JinaHub in your python codes, 
+
+```python
+from jina import Flow
+	
+f = Flow().add(
+    uses='jinahub+docker://DPRTextEncoder',
+)
+```
+
```

Comment:
do we still recommend use `jinahub`?
`from jina import Flow
	
f = Flow().add(uses='jinahub://DPRTextEncoder')`

Reply:
Good question. I would actually be in favor of removing it. I'll ask on Monday